app/models/foreman_puppet/puppet_facet_common.rb

```diff
@@ -1,8 +1,9 @@
 module ForemanPuppet
   # Common methods between host and hostgroup
   # mostly for template rendering consistency
-  module HostCommon
+  module PuppetFacetCommon
```

Comment:
While I totaly agree with this change, it should be in separate PR, right?

Reply:
opened https://github.com/theforeman/foreman_puppet/pull/158